test/playwright.spec.js

```diff
@@ -103,31 +103,29 @@ module.exports.describe = ({testRunner, product, playwrightPath}) => {
     beforeAll(async state => {
       state.browser = await browserType.launch(defaultBrowserOptions);
       state.browserServer = state.browser.__server__;
+      state._stdout = readline.createInterface({ input: state.browserServer.process().stdout });
+      state._stderr = readline.createInterface({ input: state.browserServer.process().stderr });
     });
 
     afterAll(async state => {
       await state.browserServer.close();
       state.browser = null;
       state.browserServer = null;
-    });
+      state._stdout.close();
+      state._stderr.close();
+  });
 
     beforeEach(async(state, test) => {
-      const onLine = (line) => test.output += line + '\n';
+      test.output = [];
+      const dumpout = data => test.output.push(`\x1b[33m[pw:stdio:out]\x1b[0m ${data}`);
+      const dumperr = data => test.output.push(`\x1b[31m[pw:stdio:err]\x1b[0m ${data}`);
+      state._stdout.on('line', dumpout);
```

Comment:
Why do we dump std unconditionally, but protocol only when `dumpProtocolOnFailure`?

Reply:
I'm not changing the behavior...